src/pages/dashboard/listings/[slug]/submissions.tsx

```
@@ -41,6 +40,8 @@ import { scoutsQuery } from '@/features/sponsor-dashboard/queries/scouts';
 import { submissionsQuery } from '@/features/sponsor-dashboard/queries/submissions';
 import { type ScoutRowType } from '@/features/sponsor-dashboard/types';
 
+import { WalletManagement} from '@/features/sponsor-dashboard/components/Submissions/WalletManagement';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)